src/main/java/trackbot/task/ToDo.java

```diff
@@ -0,0 +1,26 @@
+package trackbot.task;
+
+/**
+ * ToDo task with only a description.
+ */
+public class ToDo extends Task {
+
+    /**
+     * Constructs a ToDo task with the specified description.
+     *
+     * @param desc The description of the ToDo task.
+     */
+    public ToDo(String desc) {
+        super(desc);
+    }
+
+    @Override
+    public String toStorageFormat() {
```

Comment:
Great idea to have each class handle the conversion of the Task object to a String for storage in the data file. I had my main class handle this conversion which made things a bit complex.